esm-samples/jsapi-angular-cli/README.md

```diff
@@ -40,6 +40,8 @@ For additional information, see the [Build with ES modules](https://developers.a
 
 Currently, due to limitations in TypeScript, the APIs [autocasting](https://developers.arcgis.com/javascript/latest/programming-patterns/#autocasting) functionality works best in non-TypeScript applications. No changes are required if you are already using the API without any TypeScript build errors.
 
+Minimum required version is `~4.2.3`.
```

Comment:
Does it work with 4.3? If not, then maybe this should be more specific?

Reply:
Nope it doesn't. For Angular 12: TypeScript >=4.2.3 and <4.3.0. I'll update the text.